### ISSUE TITLE
Import limit?

### ISSUE THREAD

**Issue by lumninja** [sviatoslav-lebediev/anki-quizlet-importer-extended#195]
**Describe the bug**
Not sure if this is a bug or user error, but whenever I import it seems to only import 100 cards and not the full deck

**Quizlet deck URL**
https://quizlet.com/891096177/md-102-exam-flash-cards/

**Additional context**
N/A

**Comment by lumninja** [comment]
Yes that is correct.
I tried it again with a different set and it still only imported 100
![Screenshot 2024-08-25 204247](https://github.com/user-attachments/assets/e4f5be98-7055-4adf-a015-e93acee97ba5)
![Screenshot 2024-08-25 204406](https://github.com/user-attachments/assets/4c785728-f42f-45f7-b292-c3a8a6848d3a)
![Screenshot 2024-08-25 204445](https://github.com/user-attachments/assets/5df07267-9e35-451e-a8c8-d0c7ca698a3d)
![Screenshot 2024-08-25 204312](https://github.com/user-attachments/assets/e96487da-1036-4290-b300-ca217bcc6e72)

**Comment by lumninja** [comment]
**Thank you that worked! I apolozgize for the confusion.**
I did have a slight issue when it started working and I am not sure if there is a way that this could possibly be fixed by renaming every import Quizlet (NAME OF DECK). I say this because it said it successfully imported but everything merged with an existing deck. I didn't notice until I clicked browse and little did I know that every card I imported was inside that deck. Maybe that's just a me issue though. Also I don't believe this was related to the importing limit issue I was having.
